appengine-java11/springboot-helloworld/pom.xml

```diff
@@ -94,9 +94,16 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.4.1</version>
         <configuration>
-          <!-- can be set w/ -DprojectId=myProjectId on command line -->
-          <projectId>springboot-helloworld</projectId>
-          <!-- set the GAE version or use "GCLOUD_CONFIG" for an autogenerated GAE version -->
+          <!--
+            For the details of the parameters, see
+            https://cloud.google.com/appengine/docs/standard/java11/maven-reference
+          -->
+          <!-- can be set w/ -Dapp.deploy.version=myProjectId on command line -->
+          <projectId>myProjectId</projectId>
+          <!--
+            Your application's version in GAE. To use an autogenerated version, use
+            "GCLOUD_CONFIG" or leave it unspecified.
```

Comment:
My memo. The reference says we can leave it unspecified to get autogenerated version.

https://cloud.google.com/appengine/docs/standard/java11/maven-reference